data/migrations/V0087__operations_log_mv.sql

```diff
@@ -0,0 +1,51 @@
+-- Materialized View: public.ofec_operations_log_mv
+
+DROP MATERIALIZED VIEW IF EXISTS public.ofec_operations_log_mv;
+
+SET search_path = public;
+
+CREATE MATERIALIZED VIEW public.ofec_operations_log_mv AS 
+ SELECT o.sub_id,
+    o.status_num,
+    o.cand_cmte_id,
+    o.beg_image_num,
+    o.end_image_num,
+        CASE
+            WHEN f.form_tp IS NULL THEN regexp_replace(o.form_tp::text, 'A|N|T'::text, ''::text)::character varying
+            ELSE f.form_tp
+        END AS form_tp,
+    f.rpt_yr,
+    f.amndt_ind,
+    o.rpt_tp,
+    o.scan_dt,
+    o.pass_1_entry_dt,
+    o.pass_1_verified_dt,
+    o.pass_3_coding_dt,
+    o.pass_3_entry_done_dt,
+    o.receipt_dt,
+    o.beginning_coverage_dt,
+    o.ending_coverage_dt,
+    o.file_num,
+    o.prev_file_num
+   FROM staging.operations_log o
+     LEFT JOIN disclosure.f_rpt_or_form_sub f ON o.sub_id = f.sub_id
+  WHERE (o.form_tp::text <> ALL (ARRAY['RFAI'::character varying::text, 'ICI'::character varying::text, 'F1Z'::character varying::text, 'F2Z'::character varying::text])) AND o.beg_image_num IS NOT NULL AND (o.status_num = ANY (ARRAY[0::numeric, 1::numeric]))
```

Comment:
It might be more readable this way. Do we want to bring in `status_num` `0`?
```
WHERE o.form_tp NOT IN ('RFAI', 'ICI', 'F1Z', 'F2Z') 
  AND o.beg_image_num IS NOT NULL 
  AND o.status_num IN (0, 1)
```

Reply:
👍 Looks like we're bringing in the `0`'s with this migration.

Reply:
status =0 are records that not verified yet

Reply:
The original requirement is to pull records with      status_num =1 records only. 
@jwchumley is looking at the SQL.

Reply:
Can you please address https://github.com/fecgov/openFEC/pull/3203/files#r199188265?

Reply:
I'd prefer to include the zeros but that would mean we also need to be able to filter by status.